examples/siro_sandbox/sandbox_app.py

```diff
@@ -216,10 +214,18 @@ def local_end_episode(do_reset=False):
 
         self._reset_environment()
 
+    def __del__(self):
+        if self.do_network_server:
+            terminate_server_process()
```

Comment:
This looks risky. Python doesn't guarantee that `__del__` will get called before program exit, right?

Reply:
Hmm right. I'll see if I can get the client/server to misbehave in this case.